src/common/inc/types.h

```diff
@@ -68,8 +70,19 @@ typedef enum
     TRACKID_GREEN
 } TrackId;
 
-typedef struct
+typedef enum
```

Comment:
Good use of an enum.  Can you prefix each of these values with `STATIONSIDE_`, e.g. `STATIONSIDE_LEFT`?